components/collective-page/sections/Contribute.js

```diff
@@ -127,15 +132,31 @@ class SectionContribute extends React.PureComponent {
     }
   }
 
+  async handleSettingsUpdate(fn, id, settings) {
+    await fn(id, settings);
+  }
```

Comment:
What is this for?

Reply:
handleSettingsUpdate is the `GraphQL Mutation  'handleSettingsUpdate'` partially applied with lodash.

Reply:
This complexity doesn't seems necessary here, we could have the same result by simply passing a function as the prop:

```jsx
handleSettingsUpdate={settings => editCollectiveSettings(collective.id, settings)}
```

Partial saves you the ten characters of `collective.id`, but at the cost of adding 4 lines